src/hyp3_autorift/process.py

```diff
@@ -637,3 +643,7 @@ def main():
         utils.upload_file_to_s3_with_publish_access_keys(product_file, args.publish_bucket, prefix)
         utils.upload_file_to_s3_with_publish_access_keys(browse_file, args.publish_bucket, prefix)
         utils.upload_file_to_s3_with_publish_access_keys(thumbnail_file, args.publish_bucket, prefix)
+
+        if args.bucket:
+            prefix_file = save_opendata_prefix(prefix)
+            upload_file_to_s3(prefix_file, args.bucket, args.bucket_prefix)
```

Comment:
I don't love this nested `if`, but it's possible to provide only `--bucket`, only `--publish-bucket`, or both.